src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementPageEntityUpdater.java

```diff
@@ -190,11 +190,11 @@ private void updateManagementPage(OrganizationId organizationId, DefaultApi api,
   }
   
   private void updateFeaturedMedia(OrganizationId organizationId, PageId pageId, DefaultApi api, Integer featuredMedia) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "updateFeaturedMedia" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)